bin/stripdata.js

```diff
@@ -121,6 +127,31 @@ function splitDeviceId(deviceId) {
 	return retlist;
 }
 
+function removeIDsAndPayload(data) {
+	if (program.stripAll) {
+		if (data.payload)
+			delete data.payload;
+		if (data.id)
+			delete data.id;
+		if (data.guid)
+			delete data.guid;
+		if (data.bolus)
+			delete data.bolus;
+	}
+}
+
+function removeAnnotations(data) {
+	if (data.annotations)
+		delete data.annotations;
+}
+
+function stripBasalSuppressedInfo(data) {
+	if (program.stripAll) {
+		if (data.suppressed)
+			delete data.suppressed.deviceId;
```

Comment:
Do we want to delete `deviceId` outright, or only obfuscate the data (like in `stripModelAndSNForData`. For example - `basal device-Serial Number`)? Check with @HowardLook or @brandonarbiter.